osi_groundtruth.proto

```diff
@@ -70,19 +70,23 @@ message GroundTruth
     //
     repeated Lane lane = 9;
 
+    // A list of lane boundaries from lanes.
```

Comment:
In my opinion this is redundant. Just `A list of lane boundaries.` would be enough.